Rename Paperwork.getExtra() to getExtras()

egor-n opened the pull request:
Closes #2.

zsoltk commented:
Thanks for the commit! 

I originally wanted to rename only the getter, but ok, it's at least more comprehensive your way.

However, the README.md should remain the same until everything related to the next milestone is ready and the new release is published to maven central, or else someone downloading the 1.0.0 version will see a documentation that's not in sync with the actual code.

egor-n committed:
Rename Paperwork.getExtra() to getExtras()

egor-n commented:
Yeah, I didn't think about that. I reverted the `README.md` changes and rebased the commit.

I thought about renaming just the `getExtra` method, but then it wouldn't correlate with the `extra` property of the plugin, so I changed it everywhere.

zsoltk commented:
Thanks @egor-n!
